components/Buttons/src/ColorThemer/MDCButton+MDCStyle.h

```diff
@@ -0,0 +1,56 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import "MaterialButtons.h"
+
+@protocol MDCColorScheme;
+
+@interface MDCButton (MDCStyle)
+
+/**
+ Creates a new MDCButton styled as a "default size" floating action button. If @c colorScheme is
```

Comment:
Are there size numbers you can put in for default and mini sizes?

Reply:
I can add more numbers. "Default size" and "Mini size" come from material.io. They are 56 points and 40 points in diameter, respectively, with 24 by 24 point images.  "large icon" size is not in the Material spec but is 56 points in diameter with a 32 by 32 point image.

Reply:
Done